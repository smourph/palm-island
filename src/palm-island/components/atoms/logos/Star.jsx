import React from 'react';
import star from '@internal/resources/images/logo-star.png';
import './Logo.scss';

export const Star = () => (
  <span className="logo logo-common logo-star">
    <img src={star} alt="star"/>
  </span>
);

export default Star;
