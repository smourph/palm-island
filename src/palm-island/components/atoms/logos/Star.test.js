import React from 'react';
import {render, screen} from '@testing-library/react';
import Star from './Star';

it('should render snapshot', () => {
  const {asFragment} = render(<Star/>);
  expect(asFragment().firstChild).toMatchSnapshot();
});

it('should render star', () => {
  render(<Star/>);

  const icon = screen.getByAltText('star');
  expect(icon).toBeInTheDocument();
  expect(icon).toBeVisible();
  expect(icon).toHaveAttribute('src', 'logo-star.png');
});
