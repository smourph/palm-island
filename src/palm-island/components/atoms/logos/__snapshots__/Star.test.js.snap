// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render snapshot 1`] = `
<span
  class="logo logo-common logo-star"
>
  <img
    alt="star"
    src="logo-star.png"
  />
</span>
`;
